sda/cmd/s3inbox/proxy.go

```diff
@@ -280,7 +280,15 @@ func (p *Proxy) prependBucketToHostPath(r *http.Request) {
 	// Restructure request to query the users folder instead of the general bucket
 	switch r.Method {
 	case http.MethodGet:
-		if strings.Contains(r.URL.String(), "?delimiter") {
+		switch {
+		case strings.Contains(r.URL.String(), "?uploadId"):
+			// resume multipart upload
+			r.URL.Path = "/" + bucket + r.URL.Path
+		case strings.Contains(r.URL.String(), "?uploads"):
```

Comment:
This particular case seems to be missing from the `proxy_test.go` file

Reply:
Added in ec8ae55